packages/utils/package.json

```diff
@@ -66,7 +66,8 @@
   },
   "dependencies": {
     "debug": "^4.3.4",
-    "uint8arrays": "^4.0.4"
+    "uint8arrays": "^4.0.4",
+    "@waku/interfaces": "^0.0.17"
```

Comment:
is this needeD? I don't see any changes in the utils package.

Reply:
well, this was possible because `@waku/interfaces` is present in the `devDependencies` and should stay there 
don't add it to the `dependencies`